src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -53,6 +82,24 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 
             var obj = JObject.Load(reader);
             var type = Type.GetType(obj["$type"].Value<string>());
+            //if we can't find this type - try to look in our load from assemblies
+            //but only during testing.
+            if(type == null && this.isTestMode == true)
+            {
+                List<Assembly> resultList;
+
+                var typeName = obj["$type"].Value<string>().Split(',').FirstOrDefault();
+                //this assemblyName does not usually usually contain version information...
```

Comment:
duplicate of `usually`

Reply:
Can you also provide an example of assembly name so we can understand the code below better?

Reply:
assemblyName will be "CoreNodemodels"